src/hooks/lib/apr/concentrator/useConcentratorTokenVaultTotalApy.ts

```diff
@@ -0,0 +1,37 @@
+import { Address } from "wagmi"
+
+import { convertToApy } from "@/lib/api/vaults/convertToApy"
+import { useActiveChainId } from "@/hooks"
+import useConcentratorTokenVaultArbitrumTotalApr from "@/hooks/lib/apr/concentrator/useConcentratorTokenVaultArbitrumTotalApr"
+import useConcentratorTokenVaultMainnetTotalApr from "@/hooks/lib/apr/concentrator/useConcentratorTokenVaultMainnetTotalApr"
+
+export default function useConcentratorTokenVaultTotalApy({
+  asset,
+  enabled,
+}: {
+  asset: Address
+  enabled: boolean
+}) {
+  const chainId = useActiveChainId()
+  const isArbitrumFamily = chainId === 313371 || chainId === 42161
+  const tokenVaultMainnetTotalApr = useConcentratorTokenVaultMainnetTotalApr({
+    asset,
+    enabled: enabled && !isArbitrumFamily,
```

Comment:
I think you might want to explicitly check for `isMainnetFamily`. We might add more chains in the future and we should start dealing with that possibility to save us bugs/refactors in the future.